apps/src/netsim/NetSimRouterNode.js

```diff
@@ -314,11 +322,7 @@ NetSimRouterNode.create = function (shard, onComplete) {
       router.heartbeat = heartbeat;
       router.heartbeat.setBeatInterval(12000);
 
-      // Always try and update router immediately, to set its DisplayName
-      // correctly.
-      router.update(function (err) {
-        onComplete(err, router);
-      });
+      onComplete(null, router);
```

Comment:
Unrelated: This is an artifact of an old setup where router displaynames were not calculated locally from their node ID.